system/kube-monitoring/charts/prometheus-frontend/openstack-manila.alerts

```diff
@@ -0,0 +1,49 @@
+### Manila Alerts ###
+
+ALERT OpenstackManilaSharesStuck
+  IF (sum(openstack_manila_shares_stuck_count_gauge) by (host,status,check)) > 0
+  FOR 5m
+  LABELS {
+    service = "manila",
+    severity = "info",
+    tier = "openstack",
+    dashboard = "manila"
+    meta = "{{ $labels.check }}"
+    playbook = "{{.Values.ops_docu_url}}/docs/support/playbook/shares_stuck.html"
+  }
+  ANNOTATIONS {
+    summary = "Manila shares in stuck state",
+    description = "Sum of Openstack Manila Shares Stuck is more than 1",
+  }
+
+ALERT OpenstackManilaSharesStuckCreate
+  IF max(openstack_manila_shares_stuck_max_duration_gauge{status="creating"}) by (host,check) > 15
+  FOR 5m
+  LABELS {
+    service = "manila",
+    severity = "info",
+    tier = "openstack",
+    dashboard = "manila",
+    meta = "{{ $labels.check }}"
```

Comment:
I suggest meta = "{{ $value }} seconds".
Maybe this can be made more beautiful by translating it to minutes, hours .. etc. Please have a look at prometheus label replace